policies/cla.yml

```diff
@@ -106,6 +106,7 @@ configuration:
          - podman-desktop-bot
          - liurunliang-bot
          - azure-service-operator-automation[bot]
+         - azcu-helper-github-app
```

Comment:
not sure if this needs [bot] or not. 
the atual app is https://github.com/apps/azcu-helper-github-app

Reply:
needs `[bot]`